doc/compute-job-configuration.md

```diff
@@ -41,91 +41,49 @@ The main configuration files includes various global parameters used by the engi
 
-  Specify the ip address of the datastore node (running mongodb)
-
-- `mongo_port`(Required)
-
-  Specify the port number of the datastore node (running mongodb)
-
-- `mode` (Required)
-
-   The mode the engine runs. There are two available options: _cluster_ and _local_:
-
-   - If the mode is specified as _cluster_, the engine runs connecting to an existing hadoop cluster. It expects that the hadoop client is properly installed and configured.
-
-   - If the mode is specified as _local_, the engine runs local node.
-
-- `serialization` (Required)
-
-  The serialization type used. There are two available options _avro_ and _none_:
-
-  - If specified as _avro_, the engine expects metric and sync data in avro format.
-
-  - If specified as _none_, the engine expects to find metric and sync data in simple text file delimited format.
-
-- `prefilter_clean`(Required])
-
-  Boolean. Controls whether the local prefilter file will be automatically removed after it has been uploaded to the Compute Engine. If set to _true_, the local prefilter file will be automatically removed after it is  uploaded.
-
-- `sync_clean`(Required)
-
-  Boolean. Controls whether the uploaded sync files will be automatically removed after a job completion. If set to _true_ the uploaded sync files will be automatically removed after the job completion
 
 #### `[logging]`
 
 In this section we declare the specific logging options for the compute engine
 
-- `log_mode` (Required)
-
-   This parameter specifies the log_mode used by compute engine. Possible values: `syslog` (default), `file`, `none`.
-
-   - `syslog`: the compute engine is configured to use the syslog facility
-
-   - `file`: the compute engine can write directly to a file defined by `log_file`
-
-   - `none`: the compute engine does not output any logs
-
-- `log_file`
-
-  This parameter must be specified if `log_mode=file`. The file which the compute engine will use in order to write logging information
-
-- `log_level` (Required)
-
-  Possible values: `DEBUG` (default), `INFO`, `WARNING`, `ERROR`, `CRITICAL`. Defines the log level that is used by the  compute engine.
-
-- `hadoop_log_root` (Required)
-
-   Hadoop clients log level and log appender. If the user wants the hadoop components to log via SYSLOG must make sure to define an appropriate appender  in hadoop log4j.properties file. The name of this appender must be added in this parameter.
-
-   For example if the available appenders in the log4j.properties file are SYSLOG and console the above line will be:
+| Name | Type | Description | Required|
+|------|------|-------------|---------|
+|`log_mode`| String | This parameter specifies the log_mode used by the compute engine. Possible values: `syslog` (default), `file`, `none`.
+|`syslog`| String | The compute engine is configured to use the syslog facility, a) `file`: the compute engine can write directly to a file defined by `log_file`,  b) `none`: the compute engine does not output any logs| `YES`|
```

Comment:
I think this line should be part of the previous one. `log_mode` is `syslog`, `file` or `none`. It should be explained in one line.